async function authenticate (req, res, next) {
    try {
        
    } catch (error) {
        
    }
}